### PR TITLE
teams/migration: a vote for fengou1 as the reviewer

### PR DESCRIPTION
**Pull request by 3pointer**
as title.

### CODE REVIEW

**Review by YuJuncen — APPROVED** [review]
lgtm

File: votes/0620-fengou1-as-tidb-reviewer.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# A Vote for fengou1 as TiDB reviewer
+
+## Proposal
+
+[@fengou1](https://github.com/fengou1) has been has been working on `pingcap/tidb` for several months, he has made contributions in BR module and fix many bugs.
```
Comment (amyangfei) [inline votes/0620-fengou1-as-tidb-reviewer.md:5]:
```suggestion
[@fengou1](https://github.com/fengou1) has been working on `pingcap/tidb` for several months, he has made contributions in BR module and fix many bugs.
```

**Comment by sunxiaoguang** [comment]
@3pointer why this PR has not been approved by maintainers or committers in a month?

**Review by joccau — APPROVED** [review]
LGTM

**Review by kennytm — APPROVED** [review]

**Pull request merged**